src/applications/disability-benefits/wizard/pages/file-bdd.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { pageNames } from './pageList';
+import { BDD_INFO_URL } from '../../constants';
+import { BDD_FORM_ROOT_URL } from 'applications/disability-benefits/bdd/constants';
+
+const FileBDDClaim = () => (
+  <div>
```

Comment:
This wrapper could be a fragment (`<>...</>`) instead of a div.

Reply:
Oops! I was wondering why this code showed up. It's because I have this PR set up to merge into master. Good thing I just caught that. Switching to BDD branch.